app/uk/gov/hmrc/apiplatform/modules/mfa/controllers/profile/ProtectAccount.scala

```diff
@@ -38,35 +39,33 @@ import views.html.protectaccount._
 
 import scala.concurrent.{ExecutionContext, Future}
 import uk.gov.hmrc.thirdpartydeveloperfrontend.controllers.LoggedInController
-import uk.gov.hmrc.thirdpartydeveloperfrontend.controllers.Remove2SVConfirmForm
 import uk.gov.hmrc.thirdpartydeveloperfrontend.controllers.FormKeys
 
 @Singleton
 class ProtectAccount @Inject()(
-  val thirdPartyDeveloperConnector: ThirdPartyDeveloperConnector,
-  val thirdPartyDeveloperMfaConnector: ThirdPartyDeveloperMfaConnector,
-  val otpAuthUri: OtpAuthUri,
-  val mfaService: MFAService,
-  val sessionService: SessionService,
-  mcc: MessagesControllerComponents,
-  val errorHandler: ErrorHandler,
-  val mfaMandateService: MfaMandateService,
-  val cookieSigner : CookieSigner,
-  protectAccountSetupView: ProtectAccountSetupView,
-  protectedAccountView: ProtectedAccountView,
-  protectAccountView: ProtectAccountView,
-  protectAccountAccessCodeView: ProtectAccountAccessCodeView,
-  protectAccountCompletedView: ProtectAccountCompletedView,
-  protectAccountRemovalConfirmationView: ProtectAccountRemovalConfirmationView,
-  protectAccountRemovalAccessCodeView: ProtectAccountRemovalAccessCodeView,
-  protectAccountRemovalCompleteView: ProtectAccountRemovalCompleteView
+                                val thirdPartyDeveloperConnector: ThirdPartyDeveloperConnector,
+                                val thirdPartyDeveloperMfaConnector: ThirdPartyDeveloperMfaConnector,
+                                val otpAuthUri: OtpAuthUri,
+                                val mfaService: MFAService,
+                                val sessionService: SessionService,
+                                mcc: MessagesControllerComponents,
+                                val errorHandler: ErrorHandler,
+                                val mfaMandateService: MfaMandateService,
+                                val cookieSigner : CookieSigner,
+                                protectAccountSetupView: ProtectAccountSetupView,
+                                protectAccountView: ProtectAccountView,
+                                protectAccountAccessCodeView: ProtectAccountAccessCodeView,
+                                protectAccountCompletedView: ProtectAccountCompletedView,
+                                protectedAccountWithMfaDetailsView: ProtectedAccountWithMfaView,
+                                protectedAccountMfaRemovalByIdAccessCodeView: ProtectedAccountMfaRemovalByIdAccessCodeView,
+                                protectedAccountRemovalCompleteView: ProtectedAccountRemovalCompleteView
 )(
   implicit val ec: ExecutionContext,
   val appConfig: ApplicationConfig
 ) extends LoggedInController(mcc) with WithDefaultFormBinding {
 
   private val scale = 4
-  val qrCode = QRCode(scale)
+  val qrCode: QRCode = QRCode(scale)
```

Comment:
For something this obviously typed I would not add the type here.  It's a simple case class and on the same line.  Something that IntelliJ likes to do.  Tell it not to ;-)

Reply:
Done